webpack.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 
 module.exports = {
-  entry: './src/store/configureStore.js',
+	entry: './src/app.js',
```

Comment:
@jefflueck this is nitpicking but feedback you will get from most PR reviews - `entry` should be aligned vertically with `output`

Reply:
yep - `entry` should be aligned with `output` (and `module`, `devtool`, `devServer`)